tests/test_asset.py

```diff
@@ -51,12 +67,12 @@ def test_asset_update_metadata(asset_mock):
     "asset_fixture, download_url",
     [
         ("asset_mock", DOWNLOAD_URL),
-        ("asset_mock2", DOWNLOAD_URL2),
+        ("asset_mock_2", DOWNLOAD_URL2),
     ],
 )
 def test_asset_get_download_url(asset_fixture, download_url, request):
```

Comment:
The method is internal, shall not be tested explicitly, we will drop this, so let's avoid changing it